detekt-config.yml

```diff
@@ -151,24 +151,11 @@ exceptions:
   ThrowingNewInstanceOfSameException:
     active: false
   TooGenericExceptionCaught:
-    active: true
-    exceptionNames:
-      - ArrayIndexOutOfBoundsException
-      - Error
-      - Exception
-      - IllegalMonitorStateException
-      - NullPointerException
-      - IndexOutOfBoundsException
-      - RuntimeException
-      - Throwable
+    active: false
   TooGenericExceptionThrown:
-    active: true
-    exceptionNames:
-      - Error
-      - Exception
-      - NullPointerException
-      - Throwable
-      - RuntimeException
+    active: false
+    excludes:
+      - '**/gls/network/**'
```

Comment:
GLS? There are another 3 places also with GLS 😁

Reply:
Yeh I copied the rules over :D

Reply:
Removed 👍